crates/ruff_linter/src/rules/ruff/rules/unnecessary_rounding.rs

```diff
@@ -0,0 +1,186 @@
+use crate::checkers::ast::Checker;
+use ruff_diagnostics::{AlwaysFixableViolation, Applicability, Diagnostic, Edit, Fix};
+use ruff_macros::{derive_message_formats, ViolationMetadata};
+use ruff_python_ast::{Arguments, Expr, ExprCall, ExprNumberLiteral, Number};
+use ruff_python_semantic::analyze::type_inference::{NumberLike, PythonType, ResolvedPythonType};
+use ruff_python_semantic::analyze::typing;
+use ruff_text_size::Ranged;
+
+/// ## What it does
+/// Checks for `round()` calls that have no effect on the input.
+///
+/// ## Why is this bad?
+/// The unnecessary call may make the code less readable.
```

Comment:
Rounding a value that's already an integer is unnecessary. It's more clear to use the value directly.